lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -2697,18 +2697,17 @@ DefPrimitive('\renewenvironment OptionalMatch:* {}[Number][]{}{}', sub {
 # The core non-customizable part is defined here.
 # For customizable theorems, see amsthm.
 AssignValue('thm@swap' => 0);
-DefRegister('\thm@style'        => Tokenize('plain'));
-DefRegister('\thm@headfont'     => Tokens(T_CS('\bfseries')));
-DefRegister('\thm@notefont'     => Tokens(T_CS('\the'), T_CS('\thm@headfont')));
-DefRegister('\thm@bodyfont'     => Tokens(T_CS('\itshape')));
-DefRegister('\thm@headpunct'    => Tokens());
-DefRegister('\thm@styling'      => Tokens());
-DefRegister('\thm@headstyling'  => Tokens());
-DefRegister('\thm@prework'      => Tokens());
-DefRegister('\thm@postwork'     => Tokens());
-DefRegister('\thm@symbol'       => Tokens());
-DefRegister('\thm@numbering'    => Tokens(T_CS('\arabic')));
-DefRegister('\thm@headformater' => undef);
```

Comment:
Why is this definition removed?

Reply:
There is a long answer and a short answer. The short answer is that it didn't seem to be used anywhere.

The long answer is that I am quite confused by the naming and kinds of constructs for these theorem customizations. In particular, it appears that the machinery which _does_ get use is a series of assignments and lookups into state, where Tokens() get hidden behind some unfortunately named keys  -- named after command-sequences in particular. So you see both assignments and lookups to a key called "\thm@headformatter", rather than using the registers here.

I was particularly alerted by the fact that this register name had a typo (single `t` for `formater`), which was suggesting removing them may be safe. And indeed the tests passed without them.

On the other hand, I did see constructs such as  `T_CS('\the'), T_CS('\thm@headpunct')`, which were suggestive that the registers may be in use... Anyhow, it helped me with development to be sure these registers weren't playing a part in the mechanism I was changing. It would be great if I could grasp the difference...

Reply:
Oh! Didn't see the typo.  But still, because of the sneaky way registers are stored, that could have the effect of initializing it (to nothing), which apparently didn't matter. But conceivably could.  So the best solution would seem to be initializing it with the correct spelling, just in case.  :>

Reply:
Hmm, but not trace of it in texmf-dist, so maybe ok after all.